### PR TITLE
[MM-30171] Migrate changeCSS() to CSS variable in utils/utils.jsx, ln. 705

### PR DESCRIPTION
**Pull request by IanvsPoplicola**
#### Summary
Migrate a CSS variable for popover from `utils/utils.jsx` into `sass/components/_popover.scss`

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/16175

### CODE REVIEW

**Review by hmhealey — CHANGES_REQUESTED** [review]
Thanks for the PR, @IanvsPoplicola. I'm seeing some weird behaviour where the popover arrow is now visible in places that it wasn't before like above the channel members popover.
Screen Shot 2020-11-02 at 3 58 44 PM
For reference, there's no arrow on the latest release
Screen Shot 2020-11-02 at 3 59 12 PM

File: sass/components/_popover.scss [hunk 1]
```diff
@@ -66,7 +66,6 @@
         }
     }
 
-    &.bottom,
```
[the remark this answers is not in the document]
Comment (IanvsPoplicola) [reply sass/components/_popover.scss:69]: Thanks for the review! I'll look into this.

[the remark this answers is not in the document]
Comment (IanvsPoplicola) [reply sass/components/_popover.scss]: Okay I see what the problem is, I've misread the CSS and deleted the wrong stuff 🤦  Apologies! I've added it back now :)

[the remark this answers is not in the document]
Comment (IanvsPoplicola) [reply sass/components/_popover.scss]: The arrow issue's fixed on the deployed test website from what I can see!

**Review by hmhealey — APPROVED** [review]

**Review by calebroseland — APPROVED** [review]
LGTM

**Pull request merged**